ztp/gitops-subscriptions/argocd/example/optional-extra-manifest/ipsec/README.md

```diff
@@ -2,57 +2,65 @@
 
 This folder contains the files to configure IPsec encryption for external traffic
 
-* Enable IPsec encryption yaml
-* IPsec endpoint configuration with NMState yml
-* build.sh script that creates a MachineConfig to import external certs into IPsec NSS and apply the IPsec configuration
-
-Also available in OpenShift docs: [IPsec encryption for external traffic](https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn)
+Also available in OpenShift docs: [IPsec encryption for external traffic](https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn)
 
 ### Prerequisites
 
 * `butane` utility installed, minimal version 0.20.0.
+* Include `enable-ipsec.yaml` file in one of the additional install-time manifests directories defined in the `extraManifests.searchPaths` field in the SiteConfig file.
+More info about the `extraManifests.searchPaths` mechanism can be found in the [GitOps ZTP README](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/README.md)
+* Provide the following certificate files:
+  - `left_server.p12`: The certificate bundle for the IPsec endpoints
+  - `ca.pem`: The certificate authority that you signed your certificates with
 
-* IPsec endpoint certificate in PKCS#12 format and a CA cert in PEM format.
+Below are two options to configure IPsec encryption for external traffic:
+- Using NMState Operator
+- Using a MachineConfig
```

Comment:
I would reword this section so that it is clear 
we support MC for SNO and SNO+1 only
and nmstate operator for MNO only
(ie - it's not really 2 options)

Reply:
I tried to keep this open for the user to choose so I mentioned below our recommendation.  
If you think it's confusing I can change